src/Seaborn.jl

```diff
@@ -53,706 +11,77 @@ function __init__()
-* `row`, `col` : names of variables in data, optional
-    Categorical variables that will determine the faceting of the grid.
-* `col_wrap` : int, optional
-    “Wrap” the column variable at this width, so that the column facets span multiple rows. Incompatible with a row facet.
-* `row_order`, `col_order` : lists of strings, optional
-    Order to organize the rows and/or columns of the grid in, otherwise the orders are inferred from the data objects.
-* `palette` : palette name, list, or dict, optional
-    Colors to use for the different levels of the hue variable. Should be something that can be interpreted by color_palette(), or a dictionary mapping hue levels to matplotlib colors.
-* `hue_order` : list, optional
-    Specified order for the appearance of the hue variable levels, otherwise they are determined from the data. Not relevant when the hue variable is numeric.
-* `hue_norm` : tuple or Normalize object, optional
-    Normalization in data units for colormap applied to the hue variable when it is numeric. Not relevant if it is categorical.
-* `sizes` : list, dict, or tuple, optional
-    An object that determines how sizes are chosen when size is used. It can always be a list of size values or a dict mapping levels of the size variable to sizes. When size is numeric, it can also be a tuple specifying the minimum and maximum size to use such that other values are normalized within this range.
-* `size_order` : list, optional
-    Specified order for appearance of the size variable levels, otherwise they are determined from the data. Not relevant when the size variable is numeric.
-* `size_norm` : tuple or Normalize object, optional
-    Normalization in data units for scaling plot objects when the size variable is numeric.
-* `legend` : “brief”, “full”, or False, optional
-    How to draw the legend. If “brief”, numeric hue and size variables will be represented with a sample of evenly spaced values. If “full”, every group will get an entry in the legend. If False, no legend data is added and no legend is drawn.
-* `kind` : string, optional
-    Kind of plot to draw, corresponding to a seaborn relational plot. Options are {scatter and line}.
-* `height` : scalar, optional
-    Height (in inches) of each facet. See also: aspect.
-* `aspect` : scalar, optional
-    Aspect ratio of each facet, so that aspect * height gives the width of each facet in inches.
-* `facet_kws` : dict, optional
-    Dictionary of other keyword arguments to pass to FacetGrid.
-* `kwargs` : key, value pairings
-    Other keyword arguments are passed through to the underlying plotting function.
-
-# Returns:	
-
-`g` : `FacetGrid`
-    Returns the FacetGrid object with the plot on it for further tweaking.
-"""
-
-function relplot(args...; kwargs...)
-    seaborn.relplot(args...; kwargs...)
-end
-
-function load_dataset(name)
-    o = seaborn.load_dataset(name)
-    Pandas.DataFrame(o)
```

Comment:
Do you want to keep the behavior of returning a `Pandas.DataFrame`?

Reply:
(Otherwise you should remove the `Pandas` dependency.

Reply:
It's not clear to me that it's important either way...? The `PyCall.PyObject` returned by default allows access to the frame's columns and can be used in the rest of the plotting functions.

I would take a recommendation here, though.

Reply:
It would be more conservative to leave it as-is; we can always remove the Pandas.jl dependency in a different PR.